protocol/message.go

```diff
@@ -93,7 +93,7 @@ type Message struct {
 
 	// Replace indicates that this is a replacement of a message
 	// that has been updated
-	Replace   string           `json:"replace,omitEmpty"`
+	Replace   string           `json:"replace,omitempty"`
```

Comment:
Potentially breaking change cc @cammellos

Reply:
This is also likely ok